includes/Core/Modules/Tags/Module_Tag.php

```diff
@@ -27,7 +27,7 @@ abstract class Module_Tag extends Tag {
 	 * @since 1.24.0
 	 * @var string
 	 */
-	protected $slug;
+	protected $module_slug;
```

Comment:
We should update the documentation here to mention the variable name change, eg. that we removed `slug`.